examples/_Stokes_VEP_SSG/ExampleADStokes_ShearBanding.jl

```diff
@@ -0,0 +1,478 @@
+    # p2 = heatmap(xv, yv, phases.v', aspect_ratio=1, xlim=extrema(xc))
+    # display(plot(p1, p2))
+    #--------------------------------------------#
+
+    rvec = zeros(length(α))
+    err  = (x = zeros(niter), y = zeros(niter), p = zeros(niter))
+
+    to   = TimerOutput()
+
+    # TangentOperator!(𝐷, 𝐷_ctl, τ, τ0, ε̇, λ̇, η, V, Pt, Ptc, type, BC, materials, phases, Δ)
+    # ResidualContinuity2D!(R, V, Pt, Pt0, λ̇, τ0, 𝐷, phases, materials, number, type, BC, nc, Δ) 
+    # ResidualMomentum2D_x!(R, V, Pt, Pt0, λ̇, τ0, 𝐷, phases, materials, number, type, BC, nc, Δ)
+    # ResidualMomentum2D_y!(R, V, Pt, Pt0, λ̇, τ0, 𝐷, phases, materials, number, type, BC, nc, Δ)
+
+    #--------------------------------------------#
+
+    for it=1:nt
+
+        @printf("Step %04d\n", it)
+        err.x .= 0.
+        err.y .= 0.
+        err.p .= 0.
+        
+        # Swap old values 
+        τ0.xx .= τ.xx
+        τ0.yy .= τ.yy
+        τ0.xy .= τ.xy
+        Pt0   .= Pt
+
+        for iter=1:niter
+
+            @printf("Iteration %04d\n", iter)
+
+            #--------------------------------------------#
+            # Residual check        
+            @timeit to "Residual" begin
+                TangentOperator!(𝐷, 𝐷_ctl, τ, τ0, ε̇, λ̇, η, V, Pt, Ptc, type, BC, materials, phases, Δ)
+                @show extrema(λ̇.c)
+                @show extrema(λ̇.v)
+                ResidualContinuity2D!(R, V, Pt, Pt0, λ̇, τ0, 𝐷, phases, materials, number, type, BC, nc, Δ) 
+                ResidualMomentum2D_x!(R, V, Pt, Pt0, λ̇, τ0, 𝐷, phases, materials, number, type, BC, nc, Δ)
+                ResidualMomentum2D_y!(R, V, Pt, Pt0, λ̇, τ0, 𝐷, phases, materials, number, type, BC, nc, Δ)
+            end
+
+            err.x[iter] = norm(R.x[inx_Vx,iny_Vx])/sqrt(nVx)
+            err.y[iter] = norm(R.y[inx_Vy,iny_Vy])/sqrt(nVy)
+            err.p[iter] = norm(R.p[inx_c,iny_c])/sqrt(nPt)
+            max(err.x[iter], err.y[iter]) < ϵ_nl ? break : nothing
+
+            #--------------------------------------------#
+            # Set global residual vector
+            r = zeros(nVx + nVy + nPt)
+            SetRHS!(r, R, number, type, nc)
+
+            #--------------------------------------------#
+            # Assembly
+            @timeit to "Assembly" begin
+                AssembleContinuity2D!(M, V, Pt, Pt0, λ̇, τ0, 𝐷_ctl, phases, materials, number, pattern, type, BC, nc, Δ)
+                AssembleMomentum2D_x!(M, V, Pt, Pt0, λ̇, τ0, 𝐷_ctl, phases, materials, number, pattern, type, BC, nc, Δ)
+                AssembleMomentum2D_y!(M, V, Pt, Pt0, λ̇, τ0, 𝐷_ctl, phases, materials, number, pattern, type, BC, nc, Δ)
+            end
+
+            #--------------------------------------------# 
+            # Stokes operator as block matrices
+            𝐊  = [M.Vx.Vx M.Vx.Vy; M.Vy.Vx M.Vy.Vy]
+            𝐐  = [M.Vx.Pt; M.Vy.Pt]
+            𝐐ᵀ = [M.Pt.Vx M.Pt.Vy]
+            𝐏  = [M.Pt.Pt;]             
```

Comment:
```suggestion
            𝐏  = M.Pt.Pt
```
I think this is not necessary?